### PR TITLE
AttivoBotsink - fixed login and proxy handling

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23826

## Description
There was an issue with base64 encoding of the username and password in the login function.
Also added deletion of proxy as the integration does not have proxy support.

These changes were made and tested on customer instance.

### CODE REVIEW

File: Packs/AttivoBotsink/Integrations/AttivoBotsink/AttivoBotsink.py [hunk 1]
```diff
@@ -18,6 +18,11 @@
 API_PASS = demisto.params()['credentials']['password']
 VERIFY_SSL = not demisto.params()['insecure']
 
+del os.environ['HTTP_PROXY']
+del os.environ['HTTPS_PROXY']
+del os.environ['http_proxy']
+del os.environ['https_proxy']
+
```
Comment (ronykoz) [inline Packs/AttivoBotsink/Integrations/AttivoBotsink/AttivoBotsink.py:25]: change to handle_proxy

Comment (roysagi) [reply Packs/AttivoBotsink/Integrations/AttivoBotsink/AttivoBotsink.py]: done

**Comment by roysagi** [comment]
@amshamah419 - if you could please check the RN.

File: Packs/AttivoBotsink/Integrations/AttivoBotsink/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Fixed an issue where the commands did not work properly.
```
[the remark this answers is not in the document]
Comment (roysagi) [reply Packs/AttivoBotsink/Integrations/AttivoBotsink/CHANGELOG.md:2]: the actual issue was technical - there was a problem with base64 encoding and proxy handling.
But from the customer's perspective, the commands simply did not work.

[the remark this answers is not in the document]
Comment (amshamah419) [reply Packs/AttivoBotsink/Integrations/AttivoBotsink/CHANGELOG.md:2]: Okay, that makes sense. I'll approve.

**Pull request merged**